translations/messages.fr.xliff

```diff
@@ -129,7 +137,7 @@
             -->
             <trans-unit id="label.date">
                 <source>label.date</source>
-                <target>Date</target>
+                <target>Date </target>
```

Comment:
Why / where is that space needed? Probably its better to adjust the template if there needs to be an additional space. In that case its likely that every language is affected. 
Or is that space by accident?

Reply:
in French we put a space before the colon, in fact before all "double punctuation" :  ; !  ? 

I can put a &nbsp; entitie on this place

Reply:
I see the point. I think it's not as important...
I don't usually put this space before those punctuations in french but it looks better.